package-lock.json

```diff
@@ -0,0 +1,4797 @@
+{
```

Comment:
create .gitignore  in root  and add package-lock.json in it

Reply:
Sure.
But after 'npm install', in the output console it stated:
"npm notice created a lockfile as package-lock.json. You should commit this file."

I'll remove it from the pr.